transforms/universal/fdedup/python/src/get_duplicate_list_transform_local_python.py

```diff
@@ -0,0 +1,44 @@
+# (C) Copyright IBM Corp. 2024.
+# Licensed under the Apache License, Version 2.0 (the “License”);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#  http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an “AS IS” BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+################################################################################
+
+import os
+import sys
+
+from data_processing.runtime.pure_python import PythonTransformLauncher
+from data_processing.utils import ParamsUtils
+from get_duplicate_list_transform_python import (
+    GetDuplicateListPythonTransformConfiguration,
+)
+
+
+# create parameters
+input_folder = os.path.abspath(os.path.join(os.path.dirname(__file__), "..", "test-data", "expected/cluster_analysis"))
+output_folder = os.path.abspath(os.path.join(os.path.dirname(__file__), "..", "test-data", "expected"))
```

Comment:
you should never write directly to test-data.  Write somewhere else (`output` is the convention) then manually copy if needed.

Reply:
Fixed in commit https://github.com/IBM/data-prep-kit/pull/699/commits/fa5959b5f90ce90e97a52288be6aee18c06b9068